packages/react-flex/src/components/Flex/stories/FlexHorizontalConfigurable.stories.tsx

```diff
@@ -0,0 +1,389 @@
+import * as React from 'react';
+import { Flex } from '../Flex';
+import { FlexItem } from '../../FlexItem/FlexItem';
+import { FlexProps, FlexTokens } from '../Flex.types';
+import { Slider, Dropdown, TextField, Checkbox, IDropdownOption, ICheckboxStyles } from 'office-ui-fabric-react';
+
+export interface ExampleOptions {
+  numItems: number;
+  showBoxShadow: boolean;
+  preventOverflow: boolean;
+  wrap: boolean;
+  wrapperWidth: number;
+  disableShrink: boolean;
+  columnGap: number;
+  rowGap: number;
+  paddingLeft: number;
+  paddingRight: number;
+  paddingTop: number;
+  paddingBottom: number;
+  horizontalAlignment: FlexProps['horizontalAlign'];
+  verticalAlignment: FlexProps['verticalAlign'];
+  hideEmptyChildren: boolean;
+  emptyChildren: string[];
+}
+
+const FlexConfigureExampleContent: React.FunctionComponent<ExampleOptions> = props => {
+  const {
+    numItems,
+    showBoxShadow,
+    preventOverflow,
+    wrap,
+    wrapperWidth,
+    disableShrink,
+    columnGap,
+    rowGap,
+    paddingLeft,
+    paddingRight,
+    paddingTop,
+    paddingBottom,
+    horizontalAlignment,
+    verticalAlignment,
+    hideEmptyChildren,
+    emptyChildren,
+  } = props;
+  // Styles definition
+  const flexStyles = {
+    ...{
+      background: 'lightblue',
+      marginLeft: 10,
+      marginRight: 10,
+      minHeight: 100,
+      width: `calc(${wrapperWidth}% - 20px)`,
+    },
+    ...(preventOverflow
+      ? {
+          overflow: 'hidden' as 'hidden',
```

Comment:
I think you can do `const flexStyles: React.CSSProperties = {` on line 46 to get rid of this cast here.